Makefile

```diff
@@ -707,8 +707,8 @@ docs:
 .PHONY: docs-help
 docs-help: docs
 
+# Build the sphinx documentation
 docs-%:
-## Build the sphinx documentation
```

Comment:
We should spell out the doc targets we actually care about. Editing, severing etc on the docs folder should be done in the root of the docs folder.

Reply:
True, I've added that documentation here, I intend to follow this up with a PR to simplify the make file.